cmd/cli/create/create_test.go

```diff
@@ -66,6 +66,14 @@ func (s *CreateSuite) TestCreateGenericSubmitBetter() {
 				Fixture: testdata.TaskWasmJson,
 			},
 		*/
+		{
+			Name:    "docker engine spec json",
+			Fixture: testdata.JsonJobDockerEngineSpec,
+		},
+		{
+			Name:    "docker engine spec yaml",
+			Fixture: testdata.YamlJobDockerEngineSpec,
+		},
```

Comment:
Should we be doing this for WASM too?

Reply:
When we resolve this TODO, yes: https://github.com/bacalhau-project/bacalhau/blob/main/cmd/cli/create/create_test.go#L62. Unless you know of a WASM job spec that doesn't require pulling data from IPFS? 
FWIW there is test coverage for some of this in `mig_engine_test.go`

Reply:
How should we reference the wasm module from the job spec - i.e. what does the `EntryModule` of the wasm `EngineSpec` look like?

Reply:
done in https://github.com/bacalhau-project/bacalhau/pull/2706/commits/fe04d4e210eec07fd2a7fb7fc89e00e872e83056